shared/components/map.js

```diff
@@ -324,7 +322,7 @@ export default class Map extends Component {
               this.setState({markerSelect: item.key});
             }}
           >
-            {(this.state.markerSelect === item.key) ?
+            {(Platform.OS === 'ios' && this.state.markerSelect === item.key) ?

```

Comment:
If we have to implement android case, plz write to do! There are some problem in android?